os/arch/arm/src/imxrt/imxrt_lowputc.h

```diff
@@ -68,6 +68,10 @@
 #include "chip.h"
 #include "imxrt_config.h"
 
+#ifdef CONFIG_DEBUG
+#define CONFIG_DEBUG_FEATURES
```

Comment:
Could you let me know the purpose of new definition?
I think we need to use `CONFIG_DEBUG`.

Reply:
True. I will change it.